src/spanningtrees/boruvka.jl

```diff
@@ -0,0 +1,62 @@
+"""
+    boruvka_mst(g, distmx = weights(g); minimize = true)
+
+Return a tuple `(mst, weights)` where `mst` is a vector of edges representing the
+optimum (minimum, by default) spanning tree of a connected, undirected graph
+`g` with optional matrix `distmx` that provides distinct edge weights, and
+`weights` is the sum of all the edges in the solution by using
+[Boruvka's algorithm](https://en.wikipedia.org/wiki/Bor%C5%AFvka%27s_algorithm).
+The algorithm requires that all edges have different weights to correctly generate a minimun/maximum spanning tree
+### Optional Arguments
+- `minimize=true`: if set to `false`, calculate the maximum spanning tree.
+"""
+
```

Comment:
```suggestion
```
Removing newline between docstring and function

Reply:
I tried to do that and if we don't have that newline, it crashes

Reply:
this means there is a deeper problem, what error do you get?

Reply:
it looks it is something about the syntax, everybody uses that newline after that type of comment : " " " text  " " "

Reply:
Do you want me to commit it so you can take a look? I couldn't figure it out last time

Reply:
> everybody uses that newline after that type of comment

Take a look here:
https://github.com/JuliaGraphs/LightGraphs.jl/blob/master/src/graphcut/karger_min_cut.jl#L20

Note that it's not a comment but a docstring, attaching some documentation to a function

Reply:
Yeah, I'm sorry, I've used the wrong term. After each docstring they use a newline after writing "function ..."

Reply:
It is probably related to https://docs.julialang.org/en/v1/manual/documentation/index.html in Advanced Usage.

Reply:
And when I said "it crashed" I actually meant it doesn't give a positive result with the CI(travis)

Reply:
@sbromberger  Do you know something about that ?

Reply:
Nope. That sounds ... strange. Let me remove the line to see what's exactly failing.

Reply:
if you run the tests locally, can you give us the message?

Reply:
I just committed to this PR. I think (hope?) that @Lucasslimam was seeing a transient failure in one of our random tests and assuming it was due to his change. Let's see what Travis says.

Reply:
Travis is happy with the removed line.

Reply:
Oh, nice ! Sorry for the misunderstanding